app/presenters/email_alert_presenter.rb

```diff
@@ -24,6 +25,19 @@ def body
     end
   end
 
+  def tags
+    { format: document.publishing_api_document_type }.deep_merge(metadata)
+  end
+
+  def metadata
```

Comment:
Gladly, if we find a solution to this, we can also clean up the SearchPresenter and DocumentPresenter too.

I think I will use this [method](https://github.com/alphagov/specialist-publisher-rebuild/blob/master/app/models/document.rb#L92) in my first step, (although it still has the same problem of lacking clarity, I still believe "metadata" belongs in the document model rather than the presenter.)